CHANGELOG.md

```diff
@@ -1,16 +1,23 @@
 # Changelog
 
+## v0.3.0
```

Comment:
```suggestion
## main
```
or some other thing so people dont' come to our page and think a release is out?
unless you're kicking it as soon as you merge =)

(i've just been pushing release bumps straight to main when i'm ready to do it tee hee)

Reply:
Yeah I've also seen `v0.3.0-dev` in the version so it's clear that it's kind of a collection point for things. I guess either way, you have to take that step to push a commit when you do the final tagging. I'm cool with this pattern though, so we just always write our own changelog for each unreleased feature, then we cut a release separately.